react/features/toolbox/actions.native.js

```diff
@@ -203,51 +159,3 @@ export function setToolboxVisible(visible: boolean): Object {
         visible
     };
 }
-
-/**
- * Shows etherpad button if it's not shown.
- *
- * @returns {Function}
- */
-export function showEtherpadButton(): Function {
-    return (dispatch: Dispatch<*>) => {
-        dispatch(setToolbarButton('etherpad', {
-            hidden: false
-        }));
-    };
-}
-
-/**
- * Event handler for full screen toggled event.
- *
- * @param {boolean} isFullScreen - Flag showing whether app in full
- * screen mode.
- * @returns {Function}
- */
-export function toggleFullScreen(isFullScreen: boolean): Function {
-    return (dispatch: Dispatch<*>, getState: Function) => {
-        const buttonName = 'fullscreen';
-        const button = getButton(buttonName, getState());
-
-        if (button) {
-            button.toggled = isFullScreen;
-            dispatch(setToolbarButton(buttonName, button));
-        }
-    };
-}
-
-/**
- * Sets negation of button's toggle property.
- *
- * @param {string} buttonName - Button key.
- * @returns {Function}
- */
-export function toggleToolbarButton(buttonName: string): Function {
-    return (dispatch: Dispatch, getState: Function) => {
-        const button = getButton(buttonName, getState());
-
-        dispatch(setToolbarButton(buttonName, {
-            toggled: !button.toggled
-        }));
-    };
-}
```

Comment:
I get why these functions existed, to bridge a gap between the react-toolbar and the non-reduxified code, but I'm happy to see them go.